ci-operator/step-registry/cucushift/upgrade/disconnected/cucushift-upgrade-disconnected-workflow.yaml

```diff
@@ -0,0 +1,14 @@
+workflow:
+  as: cucushift-upgrade-disconnected
+  steps:
+    test:
+      - ref: cucushift-pre
+      - ref: cucushift-upgrade-pre
+      - ref: cucushift-upgrade-prehealthcheck
+      - ref: cucushift-upgrade-disconnected-mirrorimage
```

Comment:
How about adjust cucushift-upgrade-disconnected-mirrorimage and cucushift-upgrade-prehealthcheck since there maybe icmp patch which will bring mc's update? So we need wait for it.

Reply:
Good suggestion! Thanks.